qtconsole/util.py

```diff
@@ -25,7 +24,7 @@ class MetaQObjectHasTraits(MetaQObject, MetaHasTraits):
     def __new__(mcls, name, bases, classdict):
         # FIXME: this duplicates the code from MetaHasTraits.
         # I don't think a super() call will help me here.
-        for k,v in iteritems(classdict):
+        for k,v in classdict.items():
```

Comment:
This previously returned an iterator:

https://github.com/ipython/ipython_genutils/blob/7daa8b9296038f82cca7376a4e044673a115e94f/ipython_genutils/py3compat.py#L188

Please leave things as they were before.

Reply:
changes undone.